code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -75,20 +75,47 @@
 	id = "synaptizine"
 	description = "Increases resistance to stuns as well as reducing drowsiness and hallucinations."
 	color = "#C8A5DC" // rgb: 200, 165, 220
+	overdose_threshold = 40
 
 /datum/reagent/medicine/synaptizine/on_mob_life(mob/living/M)
 	M.drowsyness = max(M.drowsyness-5, 0)
 	M.AdjustParalysis(-1, 0)
 	M.AdjustStunned(-1, 0)
 	M.AdjustWeakened(-1, 0, 0)
-	if(holder.has_reagent("mindbreaker"))
-		holder.remove_reagent("mindbreaker", 5)
-	M.hallucination = max(0, M.hallucination - 10)
-	if(prob(30))
-		M.adjustToxLoss(1, 0)
+	if(prob(50))
+		M.adjustBrainLoss(-1, 0)
 		. = 1
 	..()
 
+/datum/reagent/medicine/synaptizine/overdose_process(mob/living/M)
+	if(prob(75))
+		return
+	var/effect = rand(1,15)
+	switch(volume)
+		if(1 to 39)
+			switch(effect)
+				if(1 to 7)
+					M.visible_message("<span class = 'danger'><b>[M.name]</b> suddenly and violently vomits!</span>")
+					playsound(M.loc, "sound/effects/splat.ogg", 50, 1)
+					new /obj/effect/decal/cleanable/vomit(M.loc)
```

Comment:
I dislike this cluster of lines. They're repeated many places. We don't benefit the same way from more vomit. Vomit does something here.